src/Sylius/Bundle/TaxonomyBundle/composer.json

```diff
@@ -24,6 +24,7 @@
 
         "stof/doctrine-extensions-bundle": "~1.1",
         "sylius/taxonomy": "^1.0",
+        "sylius/locale-bundle": "^1.0",
```

Comment:
Yup, TaxonomyBundle and ProductBundle are coupled with the LocaleBundle. Since I don;t remember exactly how I'll check that soon.